source/MapPanel.h

```diff
@@ -184,7 +184,8 @@ class MapPanel : public Panel {
 	public:
 		Node(const Point &position, const Color &color, const std::string &name,
 			const Color &nameColor, const Government *government)
-			: position(position), color(color), name(name), nameColor(nameColor), government(government) {}
+			: position(std::move(position)), color(std::move(color)), name(std::move(name)),
+				nameColor(std::move(nameColor)), government(government) {}
```

Comment:
It also doesn't make sense to `std::move` from these const references.
Moving from a `Point` in general is just the same as copying from it anyway.
I'm fairly sure the same will be true for `Color`.

Reply:
~~why do we already have so many moves on Point then~~